airbyte-integrations/connectors/source-google-analytics/build.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
+    id 'airbyte-docker'
+    id 'airbyte-standard-fs-source-test'
+}
+
+airbyteDocker {
+    followSymlinks = true
+}
+
+airbyteStandardFsSourceTest {
+    configPath = "secrets/config.json"
```

Comment:
Could this file be in the `priv` folder?

Reply:
we gitignore any `secrets` folder by default. If this is fine with elixir file structure, It's probably simpler to keep it in `secrets`. Otherwise we can definitely move to `priv` and gitignore that intentionally. WDYT?

Reply:
It's fine in the base path then!